### PR TITLE
Read the update 'description' from a file, not an arg

### PR DESCRIPTION
**Pull request by cjb**
r? @gabriel @keybase/updater-hackers 

This is for DESKTOP-1294, and matches up with keybase/client#3202.

### CODE REVIEW

File: update/update.go [hunk 1]
```diff
@@ -20,9 +20,8 @@ import (
 // EncodeJSON returns JSON (as bytes) for an update
 func EncodeJSON(version string, name string, description string, src string, URI fmt.Stringer, signature string) ([]byte, error) {
```
Comment (gabriel) [inline update/update.go]: Can we change `description` to `descriptionPath`

**Comment by gabriel** [comment]
🚢

File: update/update.go (hunk 1 above)
Comment (cjb) [inline update/update.go]: Would you like to change `signature` to `signaturePath` too, then, for consistency?

Comment (gabriel) [inline update/update.go]: sure

**Pull request merged**